docs/hub/audit-logs.md

```diff
@@ -6,6 +6,76 @@ This feature is part of the <a href="https://huggingface.co/enterprise" target="
 
 Audit Logs enable organization admins to easily review actions taken by members, including organization membership, repository settings and billing changes.
 
-Audit Logs are accessible through your organization admin settings.
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/audit-logs.png)
```

Comment:
missing dark mode screen

Reply:
Actually, I thought that looked better :D

Reply:
But we'll add it later